samples/msgext-northwind-inventory-python/.vscode/extensions.json

```diff
@@ -0,0 +1,6 @@
+{
+    "recommendations": [
```

Comment:
We should also include the Azure Account extension which is used to authenticate with Azure subscriptions used by TTK in actions like `arm/deploy` action.

Reply:
We're unsure about this. We compared the TTK YAML files with existing samples but couldn't find any differences. Could you please provide an example we can follow?

Reply:
@garrytrinder Could be please provide an example so that we can follow and complete this.

Reply:
@MuyangAmigo it's my understanding that TTK now uses the logged in account in the Azure Account extension for Azure tasks, should the extension be included in the recommended extensions in all TTK projects?